shared/reducers/config.js

```diff
@@ -8,6 +8,7 @@ import type {Config, GetCurrentStatusRes, ExtendedStatus} from '../constants/typ
 
 export type ConfigState = {
   globalError: ?Error,
+  daemonError: ?string,
```

Comment:
i changed this but its unclear if thats better. Errors are for runtime errors which this is not. It has a message but no filename or linenumber or stack or anything like that.